packages/domain/projet/src/lauréat/achèvement/lister/listerAchèvementEnAttente.query.ts

```diff
@@ -0,0 +1,165 @@
+import { Message, MessageHandler, mediator } from 'mediateur';
+
+import { Joined, List, RangeOptions, Where } from '@potentiel-domain/entity';
+import { DateTime } from '@potentiel-domain/common';
+import { GestionnaireRéseau } from '@potentiel-domain/reseau';
+
+import { IdentifiantProjet } from '../../..';
+import { LauréatEntity } from '../..';
+import { AchèvementEntity } from '../achèvement.entity';
+import { CandidatureEntity } from '../../../candidature';
+import { DossierRaccordementEntity, RaccordementEntity } from '../../raccordement';
+
+type AchèvementEnAttente = {
+  identifiantProjet: IdentifiantProjet.ValueType;
+  identifiantGestionnaireRéseau: GestionnaireRéseau.IdentifiantGestionnaireRéseau.ValueType;
+  referenceDossierRaccordement: string;
+  dateDCR?: DateTime.ValueType;
+  appelOffre: string;
+  codePostalInstallation: string;
```

Comment:
je sais pas si c'est la peine de mettre `installation`? je garderai le naming habituel
```suggestion
  codePostal: string;
```